README.md

```diff
@@ -44,6 +44,14 @@ Before proceeding, make sure to have the following-
 ```
 kubectl apply -f deployment.yaml
 ```
+- For a production server, we need to create a k8s `secret` to place the certificates for security purposes. 
+```
+kubectl create secret generic casbin -n default \
+  --from-file=key.pem=certs/casbin-key.pem \
+  --from-file=cert.pem=certs/casbin-crt.pem
+```
```

Comment:
This was done in order to make the middleware to work on production grade servers.